cuegui/cuegui/DarkPalette.py

```diff
@@ -35,7 +35,7 @@ def init():
     application and configures the palette and style for the Plastique
     color scheme"""
     QtGui.qApp.setPalette(DarkPalette())
-    if platform.system() in ['Darwin', 'Linux']:
+    if platform.system() in ['Darwin', 'Linux', 'Windows']:
```

Comment:
Should we just always use the style sheet at this point and get rid of the if check?